docs/source/examples.select_fps.ipynb

```diff
@@ -29,52 +29,55 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "import wfl, os, sys, yaml\n",
+    "import wfl, pathlib\n",
+    "import yaml\n",
     "from wfl.configset import ConfigSet, OutputSpec\n",
     "from wfl.descriptors.quippy import calc as calc_descriptors\n",
-    "from wfl.select.by_descriptor import greedy_fps_conf_global as get_fps\n",
+    "from wfl.select.by_descriptor import greedy_fps_conf_global\n",
     "\n",
-    "work_dir = os.path.join(os.path.dirname(wfl.__file__), \"../docs/source/examples_files/select_fps\") # Location of I/O files\n",
+    "work_dir = pathlib.Path(wfl.__file__).parents[1].joinpath(\"docs/source/examples_files/select_fps\")\n",
     "\n",
     "# Step 1: Assign descriptors to the database\n",
-    "md        = ConfigSet(os.path.join(work_dir, \"md.traj\"))\n",
-    "md_desc   = OutputSpec(files=os.path.join(work_dir, \"md_desc.xyz\"))\n",
+    "md        = ConfigSet(work_dir.joinpath(\"md.traj\"))\n",
+    "md_desc   = OutputSpec(files=work_dir.joinpath(\"md_desc.xyz\"))\n",
     "\n",
-    "with open(os.path.join(work_dir, 'params.yaml'), 'r') as foo:\n",
+    "with open(work_dir.joinpath('params.yaml'), 'r') as foo:\n",
     "    desc_dict = yaml.safe_load(foo)\n",
     "desc_dicts = [d for d in desc_dict if 'soap' in d.keys()] # filtering out only SOAP descriptor\n",
     "per_atom = True\n",
     "for param in desc_dicts:\n",
-    "    if 'average' not in param.keys():\n",
+    "    if param.pop('average', False) == False:\n",
```

Comment:
Why is there a need for an if here at all?

Reply:
Yes, indeed it can be skipped altogether. Updated now.